src/Files.App/UserControls/SideBar/SideBarView.xaml

```diff
@@ -84,13 +95,27 @@
 				</ScrollViewer>
 			</ItemsRepeaterScrollHost>
 
-			<ContentPresenter
+			<Rectangle
+				x:Name="PaneFooterDivider"
 				Grid.Row="1"
-				HorizontalAlignment="Stretch"
-				VerticalAlignment="Stretch"
-				HorizontalContentAlignment="Stretch"
-				VerticalContentAlignment="Stretch"
-				Content="{x:Bind Footer, Mode=OneWay}" />
+				Height="1"
+				Margin="4,0"
+				Fill="{ThemeResource DividerStrokeColorDefaultBrush}"
+				Visibility="Collapsed" />
+
+			<ItemsRepeater
+				x:Name="PaneFooterItemsRepeater"
+				Grid.Row="2"
+				Margin="0,4"
+				AutomationProperties.AccessibilityView="Content"
+				AutomationProperties.Name="SidebarPaneFooterItems"
+				ElementPrepared="MenuItemsHost_ElementPrepared"
+				IsTabStop="True"
+				ItemTemplate="{StaticResource DefaultSidebarItemTemplate}"
+				ItemsSource="{x:Bind PaneFooterItemSource, Mode=OneWay}"
+				VerticalCacheLength="5"
```

Comment:
I don't think cache length makes a difference here since there is only one item.

Reply:
Removed